### ISSUE TITLE
T18_3.0 Schematic

### ISSUE THREAD

**Issue by lacrimal** [LilyGO/LILYGO-T18#1]
Hi,
Can you publish schematic for T10_3.0 20190410 ?
Thanks

**Comment by immertroll** [comment]
Yeah that would be really nice. It would be also good if you could add some description.
I like to know if one pin is connected internally to measure the battery voltage a.s.o.
Thx

**Comment by vladsol** [comment]
I'm also interested :)

**Comment by kapitan-u** [comment]
Please publish schematics for T18_3.0 20190410 !
I'm wondering if there is any connections for measuring battery voltage.

**Comment by meditant** [comment]
Me too, if you can publish schematics and if it's have the same measuring battery voltage like the T8 2.2 Lion Oled

Best,

**Comment by GearWalker** [comment]
Please, release it!

**Comment by gexpo** [comment]
https://github.com/LilyGO/LILYGO-T-Energy/blob/master/WiFiScan/WiFiScan.ino

**Comment by xeijin** [comment]
I take it this isn't the schematic? (Genuine question, I am just starting out and also looking for it)

https://github.com/LilyGO/LILYGO-T-Energy/blob/master/t18_v3.0.pdf

**Comment by gexpo** [comment]
thank you.
regards.
gennaro


??

________________________________

**Comment by tedder** [comment]
[from the schematic](https://github.com/LilyGO/LILYGO-T-Energy/blob/master/t18_v3.0.pdf), battery voltage is on a voltage divider (100k/100k) on pin 35.

LED is at pin 5.

The BME280 isn't populated, but it would be at pins 20/21 for i2c.

**Comment by tedder** [comment]
and here's a pinout from lilygo.cn:
https://ae01.alicdn.com/kf/HTB1PTD2TrrpK1RjSZTEq6AWAVXay.jpg

![HTB1PTD2TrrpK1RjSZTEq6AWAVXay](https://user-images.githubusercontent.com/97914/127209894-7e530303-9dbd-47bc-8a60-6fbf8a0d33b7.jpg)
